src/android/jsc_override.cpp

```diff
@@ -143,6 +145,7 @@ static JSGlobalContextRef create_context(JSContextGroupRef group, JSClassRef glo
 
 static void swap_function()
 {
+    __android_log_print(ANDROID_LOG_INFO, "svap_function", "BEGIN");
     static int8_t s_orig_code[HOOK_SIZE];
```

Comment:
don't we need the bytes alignment `static alignas(16) s_orig_code[HOOK_SIZE];` ?

Reply:
Not sure, but I leave a TODO comment about it.